frontend/src/components/EditMarks/EditMarks.jsx

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable */
+import React, { useState } from "react";
+import { useSelector } from "react-redux";
+import { Typography , PageHeader , Input , Button , Modal , message } from "antd";
+
+import "./index.less";
+
+const EditMarks = ({ handleConfirm, handleKeyDown, inputBuffer, setInputBuffer }) => {
+
+
+  // ? Move outside of the function
+  const handleLetterGrade = (e) => {
+    setInputBuffer(e.target.value);
+    handleConfirm();
+  }
+
+  const handleInputChange = (e) => {
+    setInputBuffer(e.target.value);
+  }
+
+  return (
+    <>
+    <div className="edit-mark">
+      <div className="edit-mark-head">
+      </div>
+      <Input
+        onChange={handleInputChange}
+        onKeyDown={handleKeyDown}
+        placeholder="Enter Mark"
+      />
```

Comment:
Is it possible for this to be a `number` input component instead of allowing all inputs?

Reply:
> Is it possible for this to be a `number` input component instead of allowing all inputs?

I let it allow all inputs so that the user can also enter a letter grade like `PS`, `FL`, `DN`, `HD`;
This gets validated inside of the handleConfirm function which then chooses if we should produce a success or error message.

Reply:
Ohh alright; sounds good 👍